pallets/exchange/src/weights.rs

```diff
@@ -62,63 +60,63 @@ pub trait WeightInfo {
 pub struct HackHydraWeight<T>(PhantomData<T>);
 impl<T: frame_system::Config> WeightInfo for HackHydraWeight<T> {
 	fn known_overhead_for_on_finalize() -> Weight {
-		(26_000_000 as Weight)
+		(13_726_000 as Weight)
 			.saturating_add(T::DbWeight::get().reads(1 as Weight))
 	}
 	fn sell_intention() -> Weight {
-		(138_000_000 as Weight)
+		(85_164_000 as Weight)
 			.saturating_add(T::DbWeight::get().reads(6 as Weight))
 			.saturating_add(T::DbWeight::get().writes(2 as Weight))
 	}
 	fn buy_intention() -> Weight {
-		(138_000_000 as Weight)
+		(85_048_000 as Weight)
 			.saturating_add(T::DbWeight::get().reads(6 as Weight))
 			.saturating_add(T::DbWeight::get().writes(2 as Weight))
 	}
 	fn on_finalize(t: u32, ) -> Weight {
-		(407_248_000 as Weight)
-			// Standard Error: 394_000
-			.saturating_add((206_463_000 as Weight).saturating_mul(t as Weight))
+		(21_475_000 as Weight)
```

Comment:
WTF?

Reply:
this is actually correct now. because if you look at the other method - known_overhead_for_on_finalize - it is in the same ballpark - around 20_000_000.  Not sure why original benchmark had this huge value.  And similar values were present in lumir's benchmark on this machine ( around 20 )